tests/tests_launch/test_launch.py

```diff
@@ -332,9 +344,20 @@ def test_launch_resource_args(
     check_mock_run_info(mock_with_run_info, EMPTY_BACKEND_CONFIG, kwargs)
 
 
-def test_launch_add_base(live_mock_server):
-    queuedJob = launch_add("https://wandb.ai/mock_server_entity/tests/runs/1")
-    assert queuedJob._run_queue_item_id == "1"
+def test_launch_add_base_queued_run(live_mock_server):
+    queued_run = launch_add("https://wandb.ai/mock_server_entity/tests/runs/1")
+    assert queued_run.state == "pending"
+    assert queued_run.run_queue_item_id == "1"
+    assert queued_run.entity == "mock_server_entity"
+    assert queued_run.project == "tests"
+    assert queued_run.username == "mock_server_entity"
```

Comment:
Lol, I still don't really understand why `username` versus `entity` are different

Reply:
My understanding is that we used to allow users to refer to username, but we've switched to the entity paradigm, I can remove this check